app/views/defence_requests/show_partials/_detention_details.html.erb

```diff
@@ -0,0 +1,12 @@
+<dl class="labels">
+  <%= display_value "time_of_arrest", date_and_time_formatter(@defence_request.time_of_arrest) %>
+  <%= display_value "time_of_arrival", date_and_time_formatter(@defence_request.time_of_arrival) %>
+  <%= display_value "time_of_detention_authorised", date_and_time_formatter(@defence_request.time_of_detention_authorised) %>
```

Comment:
Datetimes should be displayed with the time first:
![screen shot 2015-06-02 at 16 59 15](https://cloud.githubusercontent.com/assets/722405/7940466/e6d9cc26-0948-11e5-90b1-73d1de50304c.png)